lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -363,7 +363,7 @@ const expectations = [
                   'type': 'node',
                   'selector': 'h3',
                   'path': '2,HTML,1,BODY,37,SECTION,1,H3',
-                  'snippet': '<h3>sub-sub-header</h3>',
```

Comment:
dont like. should we change our snippet?

Reply:
nodeLabel has it covered.